src/Database/Persist/Typed.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ImportQualifiedPost #-}
```

Comment:
Hmm, what GHC version was this introduced with? It's relatively recent IIRC and that may conflict with the bounds presented in the cabal file. I'd avoid this extension for now
```suggestion
```

Reply:
Sorry yeah, adding a compiler extension to a library is a little thoughtless. Good callout!